libpod/info.go

```diff
@@ -178,3 +186,26 @@ func (r *Runtime) GetOCIRuntimeVersion() (string, error) {
 	}
 	return strings.TrimSuffix(output, "\n"), nil
 }
+
+// GetHostDistributionInfo returns a map containing the host's distribution and version
+func (r *Runtime) GetHostDistributionInfo() map[string]string {
+	dist := make(map[string]string)
+
+	f, err := os.Open("/etc/os-release")
+	defer f.Close()
+	if err != nil {
+		return dist
+	}
+
+	l := bufio.NewScanner(f)
+
+	for l.Scan() {
+		if strings.HasPrefix(l.Text(), "ID=") {
+			dist["Distribution"] = strings.Replace(strings.TrimSpace(strings.Join(strings.Split(l.Text(), "=")[1:], "")), "\"", "", -1)
+		}
+		if strings.HasPrefix(l.Text(), "VERSION_ID=") {
+			dist["Version"] = strings.Replace(strings.TrimSpace(strings.Join(strings.Split(l.Text(), "=")[1:], "")), "\"", "", -1)
```

Comment:
Same as above. Also, this probably won't work on a bunch of distros - Debian doesn't have a VERSION_ID.

Reply:
```± |initial ?:1 ✗| → sudo podman run -it --rm debian cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

Reply:
Neither does Arch

Reply:
Ah, I'm on Sid - that's probably why.

I suppose version doesn't make much sense on rolling release distros like debian unstable/Arch/OpenSUSE Tumbleweed

Reply:
Maybe we should populate this with "Unknown" by default, so if we don't find/set it here we get something out?

Reply:
i checked the major distributions, seems they all adhere to it

Reply:
sure, thats good reasoning